transform/operations.go

```diff
@@ -347,12 +347,12 @@ func (c *stringToFloat64) init(args map[string]string) error {
 func (c *stringToFloat64) transform(raw interface{}) (interface{}, error) {
 	in, ok := raw.(string)
 	if !ok {
-		return nil, fmt.Errorf("stringToFloat64 only supports strings")
+		return nil, errors.New("stringToFloat64 only supports strings")
```

Comment:
Change this one to `fmt.Errorf("stringToFloat64 only supports strings, raw type: %T, raw")` or something like that.